### DISCUSSION TITLE
sensor.victron_today_kwh can't be used in HA enery dashboard

### DISCUSSION THREAD

**Discussion by mspcoding** [softwarecrash/Victron2MQTT#9] [General]
Currently sensor victron_today_kwh or victron_total_kwh can't be used as solar input in HA energy dashboard. Both are not listed in the selection box. 

The sensor gets defined by HA Discovery in Victron2MQTT .

Current status attributes:
```
unit_of_measurement: kWh
device_class: energy
icon: mdi:solar-power
friendly_name: PV Dach Energie
```

It seems that either
`state_class: total `
or
`state_class: total_increasing`
is missing here.

**Comment by Jarnsen** [comment]
Ja das ist richtig, ist in Arbeit. Kannst den Punkt aber auch selbst mit dem MQTT-Explorer hinzufügen. Weißt du wie das geht? Oder brauchst Hilfe?

**Comment by mspcoding** [comment]
Danke für den Hinweis. Habe es im MQTT-Explorer beim Attribut definiert und es funktioniert solange der MQTT-Explorer geöffnet ist. Nach gewisser Zeit wird das zusätzliche Attribut wieder entfernt.

**Comment by softwarecrash** [comment]
@mspcoding 
probier mal diese version
[Victron2MQTT_d1_mini_1.1.4.zip](https://github.com/softwarecrash/Victron2MQTT/files/14549424/Victron2MQTT_d1_mini_1.1.4.zip)

**Reply by mspcoding** [reply]
Funktioniert leider nur kurz. Der Victron 150/35 wird deaktiviert mit Off_reason=Remote input und Operation_state=Off

Stecke ich die VE Direkt Verbindung zum Victron3MQTT ab läuft er sofort wieder.

**Reply by mspcoding** [reply]
Hm das kann auch mit dem alternativen Ladegerät zusammenhängen, wenn das an geht schaltet der Victron ab.

**Reply by mspcoding** [reply]
Ja das alternative Ladegerät hatte einen zu hohen Volt Wert eingestellt gehabt. Jetzt hab ich es auf eine niedrige Spannung eingestellt und der Victron schaltet nicht mehr ab. Somit funktioniert jetzt alles wie gewünscht und die HA Integration passt nun auch. Vielen Dank für das schnelle FW Update.
